src/error.ts

```diff
@@ -132,7 +142,13 @@ export class MongoError extends Error {
 
   constructor(message: string | Error) {
     if (message instanceof Error) {
-      super(message.message);
+      if (isAggregateError(message)) {
+        const combinedMessage = 'Aggregate Error: ' + message.errors.map(e => e.message).join(', ');
+        super(combinedMessage);
```

Comment:
```suggestion
        super(message.errors.map(({ message }) => message).join(', '));
```

Is indication of "Aggregate Error" important? I would expect us to have the error messages directly